collections/impact-observatory-lulc-map.yaml

```diff
@@ -0,0 +1,46 @@
+Name: 10m Annual Land Use Land Cover (9-class) 
+Description: |
+  The [10m Annual Land Use Land Cover (LULC)](https://www.impactobservatory.com/global_maps/) map is produced Impact Observatocy, Microsoft, and Esri collaboratively.
+  The data collection is derived from ESA Sentinel-2 imagery at 10m resolution globaly using Impact Ovservatory's state of the art deep learning AI land classification model which is trained by billions of human-labeled image pixels.
+  There are 9 LULC classes generated by the algorithm, including Built, Crops, Trees, Water, Rangeland, Flooded Vegetation, Snow/Ice, Bare Ground, and Clouds.
+
+AdditionalInfoExternal:
+    Title: Additional info
+    Path: impact-observatory-lulc-map/README.MD 
+Image: impact-observatory-lulc-map/io-lulc-map.png
+# ViewOnEOBrowser: 
+Resolution: 10m
+GeographicalCoverage: Global land area
+TemporalAvailability: 2017, 2018, 2019, 2020, 2021
+UpdateFrequency: Annually, each January
+BandInformation: 
+  Table:
+      Columns:
+        - Name: Name
+          Title: Name
+        - Name: Description
+          Title: Description
+      Rows:
+        - Name: lulc
+          Description: Main discrete classification defined in the [product documentation](https://www.impactobservatory.com/global_maps/).
```

Comment:
Within a table, markdown structure for links does not work, 

Html structure works

`<a href="https://www.impactobservatory.com/global_maps/">product documentation</a>`